projects/libpsl/build.sh

```diff
@@ -17,6 +17,7 @@
 
 # avoid iconv() memleak on Ubuntu 16.04 image (breaks test suite)
 export ASAN_OPTIONS=detect_leaks=0
+export LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
This setting will be used only during the build time, not during the execution time. 
Is that what you expect?